src/SprykerEco/Zed/ProductManagementAi/Presentation/_partials/category-modal.twig

```diff
@@ -0,0 +1,54 @@
+{% block body %}
+	<dialog popover id="ai-category-modal" class="ai-category-modal modal-popover">
+		{% block modal %}
+			<div class="modal">
+				<div class="modal-dialog">
+					<div class="modal-content">
+						{% block header %}
+							<div class="modal-header">
+								<button type="button" class="close" popovertarget="ai-category-modal" aria-label="Close">
+									<span aria-hidden="true">&times;</span>
+								</button>
+								<h4 class="modal-title">{{ 'AI assistant' | trans }}</h4>
+							</div>
+						{% endblock %}
+
+						{% block content %}
+							<div class="modal-body js-modal-body">
+								{% block loading %}
+									<div class="ai-category-modal__loading">
+										<span>{{ 'Looking for categories' | trans }}</span>
+										<span class="fa fa-spinner fa-spin"></span>
+									</div>
+								{% endblock %}
+
+								{% block empty %}
+									<span class="text-center ai-category-modal__empty">{{ 'Please fill in the product name and description for at least one locale' | trans }}</span>
+								{% endblock %}
+
+								{% block preview %}
+									<div class="ai-category-modal__select">
+										<select multiple dropdown-parent=".js-modal-body" class="js-ai-category-select spryker-form-select2combobox"></select>
```

Comment:
I think we are not useing select2 anymore, do we still need `spryker-form-select2combobox`?